css/collapsible_widget.css

```diff
@@ -0,0 +1,83 @@
+.collapsible-actions {
```

Comment:
The buttons seem smaller than the rest of the similar buttons in the form and a bit misaligned, although checking on paragraphs they seem fine (they are however misaligned in paragraphs as well). The screenshot is from the Policy CT. Just a suggestion but it might be worth checking.
<img width="1027" alt="operation buttons" src="https://github.com/user-attachments/assets/f1c63512-4330-4d7d-b5f7-1f279906b212" />

Reply:
Now I get this:
<img width="1000" alt="timeline buttons" src="https://github.com/user-attachments/assets/9a85bf0a-e36a-4a2e-bcdd-ec86b3a561dc" />

Reply:
Nvm, it's actually fixed.